srv/wordpress/wp-content/plugins/acore-wp-plugin/src/components/UserPanel/User.controller.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace ACore;
+
+require_once 'User.view.php';
+
+class UserController {
+
+    /**
+     *
+     * @var UserView
+     */
+    private $view;
+
+    public function __construct() {
+        $this->view = new UserView($this);
+    }
+
+    public function showRafProgress() {
+        try {
+            $acServices = ACoreServices::I();
+        } catch (\Exception $e) {
+            wp_die(__($e->getMessage()));
+        }
+        $user = wp_get_current_user();
+
+        if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+            $maxRecruitDatetime = (new \DateTime($user->get("user_registered")))->modify('+7days');
+            if ($maxRecruitDatetime >= (new \DateTime())) {
+                ?><div class="notice notice-error">
+                    <p>You can't be recruited by a friend, the 7 days limit are passed.</p>

```

Comment:
7 days after registration of 7 days of playing?
We could consider the playing time

Reply:
I don't think 7days of playing time is a good idea, it could be after a month of the creation date of the account.